src/main/resources/templates/courselist.twig

```diff
@@ -0,0 +1,10 @@
+<link href="/css/courselist.css" rel="stylesheet"/>
+
+{% block courselist %}
+<div class="list-group courselist">
+    <p class="list-group-item active">Course List</p>
+    {% for courseItem in courseList %}
+        <a href="#" class="list-group-item courselist-item">{{ courseItem.getName() }}</a>
+    {% endfor %}
+</div>
+{% endblock %}
```

Comment:
no line at eof

Reply:
Do you want a newline or want it removed? :)

Reply:
Whoops, should've been more clear. There should be a newline at eof (https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline/729795#729795)

Reply:
Ooh ok I see. I just pulled down this file, and it is newline terminated.